examples/swarmion-with-cdk/backend/core/package.json

```diff
@@ -36,6 +38,7 @@
     "dotenv": "^16.0.3",
     "esbuild": "^0.15.14",
     "eslint": "^8.27.0",
+    "json-schema-to-ts": "^2.6.0",
```

Comment:
you shouldn't need this if you bump the `@swarmion/serverless-contracts` package